### ISSUE TITLE
watching 9000+ files - this might cause high cpu usage on git-bash 2

### ISSUE THREAD

**Issue by Mosho1** [remy/nodemon#624]
I'm using a watch on a single file and am getting this message. Using the latest version on windows.

    [nodemon] watching: gulpfile.babel.js

**Comment by harriha** [comment]
I'm having such a problem on Windows 7 as well (node version: 0.12.7, nodemon: 1.4.1).

Nodemon seems to read my `nodemon.json` fine, but no matter how I modify the `ignore` and/or `watch` values, always the same thing, it watches my `node_modules` always. I'm giving, for instance, `"watch": ["*.foo"]`, which is reflected to the verbose output, but still I see `watching 38,915 files` with that warning. Tried with explicit and default ignore rules, no change.

Not seeing the same with Windows 8.1, node 0.12.0 and nodemon 1.4.1. Although, here I'm seeing `watching 0 files` even though that's not true.

Hmm, noticed that if I run the nodemon from Windows default command prompt on that problematic Windows 7 machine, things work as they do on that Windows 8.1 machine of mine, seeing `watching 0 files`. 

The Git/Git bash on the Win7 machine is the latest brand new 2.5.0, whereas on my Win8 machine I'm using an older version. Could the shell change the behavior like this? I chose the option "Use Windows' default console window" when installing that git 2.5.0, and it seemed to me that there's been some changes in that front in the git bash package.
